support-frontend/assets/pages/paper-subscription-checkout/components/orderSummary/orderSummary.jsx

```diff
@@ -63,14 +63,24 @@ function PaperOrderSummary(props: PropTypes) {
       paperProductsWithoutDigital[props.productOption],
     )
     : props.total;
-
+  // This allows us to get the price without promotion, so we can say what the price will revert to
+  const paperWithoutPromo = getProductPrice(
+    props.productPrices,
+    props.fulfilmentOption,
+    props.productOption,
+  );
+  const monthsDiscounted = paperWithoutPromo.promotions &&
+    paperWithoutPromo.promotions.length && paperWithoutPromo.promotions[0].numberOfDiscountedPeriods;
+  const priceHasPromo = paperWithoutPromo.promotions && paperWithoutPromo.promotions.length > 0;
+  const promotionPriceString = priceHasPromo && monthsDiscounted
+    ? ` for ${monthsDiscounted} months, then ${showPrice(paperWithoutPromo, false)} per month` : '';
```

Comment:
```suggestion
  const activePromo = getAppliedPromo(paperWithoutPromo.promotions);
  const monthsDiscounted = activePromo && activePromo.numberOfDiscountedPeriods;
  const promotionPriceString = activePromo && monthsDiscounted
    ? ` for ${monthsDiscounted} months, then ${showPrice(paperWithoutPromo, false)} per month` : '';
```

Just a couple of suggestions to tidy up this logic a bit- the getAppliedPromo function already discerns whether there's a relevant promotion that we can get information from.

Reply:
Oh whoops I was too slow 😅

Reply:
Thanks for this - I'm going to do a tiny PR to fix this - coming your way shortly!